Bindings/Java/Matlab/Hopper_Device/BuildHopper.m

```diff
@@ -49,22 +49,25 @@
 
 p = inputParser();
 defaultMaxIsometricForce = 4000.0;
-defaultMillardTendonParams = [0.049 28.1 0.67 0.5 0.25];
+defaultOptimalFiberLength = 0.55;
+defaultTendonSlackLength = 0.25;
```

Comment:
Hmm... I think it's fine for the parameters to differ, and if we change the parameters, we would need to readjust other parameters to get the initial model to not jump.